attest/enclave-api/src/lib.rs

```diff
@@ -87,6 +87,19 @@ pub struct SealedClientMessage {
     pub data: IntelSealed,
 }
 
+/// SealedClientRequest structure, which is used in the enclave during the Intel
+/// sealing process. Ensures that the data being passed to Intel is not empty.
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct SealedClientRequest {
```

Comment:
I don't like the name `SealedClientRequest`, as it's specifically _not_ sealed at this point. Maybe something like `ClientSealingRequest` or just `SealRequest`?

Reply:
Good point, I'd vote for `UnsealedClientRequest` or `PlaintextClientRequest` as the way to go, TBH.

Reply:
Done.